spec/sidekiq/form526_submission_failure_email_job_spec.rb

```diff
@@ -0,0 +1,151 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Form526SubmissionFailureEmailJob, type: :job do
+  subject { described_class }
+
+  let!(:form526_submission) { create(:form526_submission) }
+  let(:email_service) { double('VaNotify::Service') }
+
+  before do
+    Sidekiq::Job.clear_all
+    allow(VaNotify::Service)
+      .to receive(:new)
+      .with(Settings.vanotify.services.benefits_disability.api_key)
+      .and_return(email_service)
+  end
+
+  describe '#perform' do
+    let(:expected_params) do
+      {
+        email_address: 'test@email.com',
+        template_id: 'form526_submission_failure_notification_template_id',
+        personalisation: {
+          first_name: form526_submission.get_first_name,
+          date_submitted: form526_submission.format_creation_time_for_mailers
+        }
+      }
+    end
+
+    it 'dispatches a failure notification email with an obscured filename' do
```

Comment:
is the `obscured filename` stuff present in this PR? I dont see any like.... filenames param being sent along with the `send_email` call, but maybe that is happening elsewhere somehow, not sure how though.

Reply:
Do we have the template from research yet? I thought we were going to need to pass along those filenames.... or we just going to add that later?

Reply:
i was given the template ID for staging, which is enough to itterate.